experimental/generator-dotnet-yeoman/generators/app/templates/functions/Dialogs/echo/echo.dialog

```diff
@@ -0,0 +1,68 @@
+{
+  "$kind": "Microsoft.AdaptiveDialog",
+  "$designer": {
+    "id": "433224",
```

Comment:
Not sure how ID generation works in Composer, or indeed how much it matters that we modify it, but we should consider if this is something we need to generate as we do with sln/csproj IDs.

Reply:
Dialog assets removed.